include/powsybl/iidm/util/ConnectedComponents.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_UTIL_CONNECTEDCOMPONENTS_HPP
+#define POWSYBL_IIDM_UTIL_CONNECTEDCOMPONENTS_HPP
+
+#include <powsybl/stdcxx/optional.hpp>
+#include <powsybl/stdcxx/reference_wrapper.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+class Bus;
+
+namespace ConnectedComponents {
```

Comment:
This is not related to this PR, but classes/methods/functions in the "util" directory does not always are in namespace `powsybl::iidm::util`. I think either all or none of them should be.

Reply:
https://github.com/powsybl/powsybl-iidm4cpp/issues/222